### PR TITLE
Update release process

### PR DESCRIPTION
**Pull request by viirya**
`HiveExternalCatalogVersionsSuite` now doesn't need to manually update for `testingVersions`. It automatically gets the latest releases now.

### CODE REVIEW

**Comment by viirya** [comment]
cc @dongjoon-hyun @srowen @HyukjinKwon

File: site/sitemap.xml [hunk 1]
```diff
@@ -876,27 +876,27 @@
   <changefreq>weekly</changefreq>
 </url>
 <url>
-  <loc>https://spark.apache.org/screencasts/</loc>
+  <loc>https://spark.apache.org/graphx/</loc>
```
Comment (srowen) [inline site/sitemap.xml]: You could revert this change, but I'm not sure which version is 'right' - which one is the result as of the latest site generation tools. Rest is OK.

Comment (dongjoon-hyun) [reply site/sitemap.xml]: Yes, +1 for reverting this file change.

**Comment by dongjoon-hyun** [comment]
+1, LGTM (except the above comment on site/sitemap.xml).

File: site/sitemap.xml (hunk 1 above)
Comment (viirya) [reply site/sitemap.xml]: Yea, let me revert it.

**Review by dongjoon-hyun — APPROVED** [review]

**Comment by maropu** [comment]
lgtm, too.

**Comment by viirya** [comment]
Thanks @dongjoon-hyun @srowen @maropu

**Comment by HyukjinKwon** [comment]
👏

**Comment by viirya** [comment]
Thank you @HyukjinKwon
